ebpf/xdp_filter.c

```diff
@@ -42,6 +42,13 @@
 /* Increase CPUMAP_MAX_CPUS if ever you have more than 64 CPUs */
 #define CPUMAP_MAX_CPUS     64
 
+#define USE_PERCPU_HASH    1
+#define GOT_TX_PEER	1
```

Comment:
what is this?

Reply:
In current kernel, Netronome cards are not supporting the in-XDP transmit to another interface. As this is a specific map type, this triggers a failure of eBPF verifier. So workaround is to set this to 0 for Netronome harware bypass.

Reply:
A small code comment to explain the usage would be good.

Reply:
Done.